.github/workflows/submitty_ci.yml

```diff
@@ -305,6 +305,15 @@ jobs:
           - 5432:5432
 
     steps:
+      - name: Delete huge unnecessary folders, and clean apt cache
+        run: |
+          rm -rf /opt/hostedtoolcache
+          rm -rf /usr/share/dotnet
+          rm -rf /opt/ghc
+          rm -rf "/usr/local/share/boost"
+          rm -rf "$AGENT_TOOLSDIRECTORY"
+          sudo apt-get clean
+
```

Comment:
Instead of duplicating this code, is there any reason why you can't move this to the setup composite action?

Reply:
There is not, it has been moved to composite.